admin/src/templates/index.html

```diff
@@ -6,13 +6,7 @@
     <title>Medic Mobile administration console</title>
   </head>
   <body ng-controller="MainCtrl">
-      <nav class="header">
-        <div class="container">
-          <div class="navbar-header">
-            <h1 translate>admin.app.name</h1>
-          </div>
-        </div>
-      </nav>
+      <ng-include src="'templates/header.html'"></ng-include>
```

Comment:
Good idea to break this out as a separate include.

The indentation is a little off though.